config/plugin-commands-config/src/configGet.ts

```diff
@@ -9,5 +9,13 @@ export function configGet (opts: ConfigCommandOptions, key: string): { output: s
     return { output: '', exitCode: exitCode ?? 0 }
   }
   const config = opts.rawConfig[kebabCase(key)]
-  return { output: Array.isArray(config) ? config.join(',') : String(config), exitCode: 0 }
+  let output: string
+  if (Array.isArray(config)) {
+    output = config.join(',')
```

Comment:
This was on me: https://github.com/pnpm/pnpm/pull/7917. I don't remember why I decided to return a comma-separated list instead of a JSON string, but changing it now would be a breaking change.